engines/instance_verification/spec/requests/api/connect/v3/systems/products_controller_spec.rb

```diff
@@ -315,7 +315,7 @@
           let(:scc_annouce_body) do
             {
               hostname: system.hostname,
-              hwinfo: JSON.parse(system.system_information).merge({ instance_data: system.instance_data }),
+              hwinfo: JSON.parse(system.system_information),
```

Comment:
Why does this test not send instance data anymore?

Reply:
Because it is the correct behavior

As explained on [this comment](https://github.com/SUSE/rmt/pull/1216#discussion_r1732411906) the instance metadata 
(to announce a PAYG system that maybe HYBRID, because it is activating a non free extension) 
comes from the [system](https://github.com/SUSE/rmt/blob/master/engines/scc_proxy/lib/scc_proxy/engine.rb#L332) not the params

Reply:
The test still checks the instance metadata is used in `InstanceVerification` check [here](https://github.com/SUSE/rmt/blob/master/engines/instance_verification/spec/requests/api/connect/v3/systems/products_controller_spec.rb#L336)

Reply:
But this is the announce payload, not the payload to activate a product. I thought the announce payload from suseconnect includes the instance data?

Reply:
the params on [this request](https://github.com/SUSE/rmt/blob/master/engines/scc_proxy/lib/scc_proxy/engine.rb#L312)  when activating a product to SCC does not include instance data
  
  but [RMT does not send instance data](https://github.com/SUSE/rmt/blob/master/engines/scc_proxy/lib/scc_proxy/engine.rb#L76) to SCC ? SCC does not care about the instance data, does it ?
  
RMT did not send instance data before this change to announce the system
  
 ```ruby
# code before this change
hw_info_keys = %i[cpus sockets hypervisor arch uuid cloud_provider]
hw_info = params['hwinfo'].symbolize_keys.slice(*hw_info_keys)
scc_request.body = {
  hostname: params['hostname'],
  hwinfo: hw_info,
  byos: @instance_id
}.to_json
```